toolchain/templates/include/helpers.mako

```diff
@@ -64,13 +64,13 @@ fi
 
 cd "${os.path.dirname(input)}"
 
-t_${target.name}_start=$(date +%s)
+t_${target.name}_start=$(date +%s.%N)
```

Comment:
Note that on Mac:

```console
lawn-10-91-109-237: /Users/spencer $ date +%s.%N
1715972166.N
lawn-10-91-109-237: /Users/spencer $ gdate +%s.%N
1715972183.446123000
```

So, benchmarking will not work on MacOS unless `gdate` (gnu date) is installed and aliased to `date`.

Reply:
Is this something that needs fixed?

Reply:
I found an easy fix. `perl -MTime::HiRes=time -e 'printf "%.9f\n", time'`

Reply:
perl!

Reply:
presumably we woud like to be able to run benchmarking on macos so yeah the other option is better (there are many options, it's just a matter of getting it working).

Reply:
Can't you do this?
```python
python3 -c 'import time; print(time.time())'
```
We require Python be installed so this seems like the cleanest solution.

Reply:
You can also subtract the two time points in Python. Like this: 
```python
python3 -c "print($t_${target.name}_stop - $t_${target.name}_start)"
```
Admittedly, you could replace `$t_${target.name}_stop` with `time.time()` if you import `time`.

Reply:
It seems reasonable to exclude the startup costs of modules and such and do this in the batch script. The startup times to load modules can vary by several seconds sometimes

Reply:
indeed